UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
```

Comment:
Could we link to an example? I'm familiar with what this means in the context of e.g. a dialog box, but not in the context of a content-centric page.